bin/proving-service/.env

```diff
@@ -0,0 +1,16 @@
+PROXY_HOST="0.0.0.0"
+PROXY_PORT="8082"
+PROXY_WORKERS_UPDATE_PORT="8083"
+PROXY_TIMEOUT_SECS="100"
+PROXY_CONNECTION_TIMEOUT_SECS="10"
+PROXY_MAX_QUEUE_ITEMS="10"
+PROXY_MAX_RETRIES_PER_REQUEST="1"
+PROXY_MAX_REQ_PER_SEC="5"
+PROXY_AVAILABLE_WORKERS_POLLING_TIME_MS="20"
+PROXY_HEALTH_CHECK_INTERVAL_SECS="1"
+PROXY_PROMETHEUS_HOST="127.0.0.1"
+PROXY_PROMETHEUS_PORT="6192"
+RUST_LOG="info"
+WORKER_HOST="0.0.0.0"
+WORKER_PORT="8084"
+WORKERS="0.0.0.0:8084,0.0.0.0:50051"
```

Comment:
I would probably remove `WORKERS` from here. I think these should be provided via command line (though, I could be convinced otherwise).

Also, I think we should probably have a more unique prefix (`PROXY` and `WORKER` don't sound too unique). Maybe we could use `MPS` (for Miden proving service) - e.g., `MPS_HOST`, `MPS_PORT`, `MPS_WORKER_HOST`).